INSTALL

```diff
@@ -473,6 +480,9 @@ The specific libraries and their options are:
   URL: https://zlib.net
   Options: ZLIB_OPT, ZLIB_LIB.
 
+- PUMI (optional), used when MFEM_USE_PUMI = YES.
+  URL: https://scorec.rpi.edu/pumi
```

Comment:
Shouldn't there be `PUMI_OPT` and `PUMI_LIB` variables? How does the regular make work?

Reply:
This merge doesn't support gnu-make builds; just cmake.  We checked to see that we did not break the mfem gnu-make build (without pumi).